Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1354,9 +1354,18 @@
   description: uplink-soap-desc
   productEntity: SoapSyndie
   cost:
-    Telecrystal: 2 # It used to be 1, but possibility of unfair use.
+    Telecrystal: 1
   categories:
   - UplinkMisc
+  conditions: # Operatives are not slippery guys. They're just dangerous.
+  - !type:StoreWhitelistCondition
+    blacklist:
+      tags:
+      - NukeOpsUplink
+  - !type:ListingLimitedStockCondition
+    stock: 1
```

Comment:
since nukies arent allowed to buy it remove limited stock

Reply:
> since nukies arent allowed to buy it remove limited stock

I think it's fair that operatives can't buy it and traitors are limited to 1 :trollface:

Reply:
ruined soap dealer rp buying 20 soap and giving it all to the mime

Reply:
> ruined soap dealer rp buying 20 soap and giving it all to the mime

:trollface: 
In fact, I've written before that I think soap is too strong an item without risk. If you limit it to 1 - you get it stolen and that's it, you don't have a cool stun on the cheap.